src/sdk/filters/makeControllers.js

```diff
@@ -23,6 +23,7 @@ export default chart => {
     }
     const attributes = getInitialFilterAttributes(chart)
     chart.updateAttributes(attributes)
+    chart.updateAttribute("selectedDimensions", null)
```

Comment:
Why don't you add it on the above object? `getInitialFilterAttributes` ?

Reply:
Seems that `getInitialFilterAttributes` is being used elsewhere too, inside our fetch mechanism